apache-rat-core/src/main/resources/org/apache/rat/plain-rat.xsl

```diff
@@ -83,18 +118,74 @@ Archives:
     U       Unknown file.
   
 </xsl:text>
- <xsl:for-each select='descendant::resource'>
-  <xsl:choose>
-     <xsl:when test='license/@approval="false"'><xsl:value-of select='concat($newline, "!")'/></xsl:when>
-     <xsl:otherwise><xsl:value-of select='concat($newline, " ")'/></xsl:otherwise>
-   </xsl:choose>
-   <xsl:value-of select='concat(substring(@type, 1, 1), " ", @name)'/>
-   <xsl:for-each select='descendant::license'>
-       <xsl:value-of select='concat($newline, "    ", substring(concat(@family, "     "), 1, 5),
-        "    ", substring(concat(@id, "          "), 1,10), "    ", @name)'/>
-        <xsl:if test="@approval='false'"> (Unapproved)</xsl:if>
-   </xsl:for-each>
- </xsl:for-each>
-*****************************************************
-</xsl:template>
+        <xsl:for-each select='descendant::resource'>
+            <xsl:call-template name="document">
+                <xsl:with-param name="name"><xsl:value-of select='@name'/></xsl:with-param>
+                <xsl:with-param name="mediaType"><xsl:value-of select='@mediaType'/></xsl:with-param>
+                <xsl:with-param name="encoding"><xsl:value-of select='@encoding'/></xsl:with-param>
+                <xsl:with-param name="type"><xsl:value-of select="substring(@type, 1, 1)"/></xsl:with-param>
+                <xsl:with-param name="leadin"><xsl:choose>
```

Comment:
same as above? Does it mean leading? If so - should it be called 'prefix' instead?